module/Photo/view/photo/photo/view.phtml

```diff
@@ -5,56 +5,66 @@ $this->scriptUrl()->requireUrl('member/search');
 ?>
 <?php if ($photo != null): ?>
     <div clas="row">
+        <h1>Foto's</h1>
+        
         <nav>
             <ul class="pager">
+                <li>
+                    <a href="<?= $this->url('photo/album', array('album_id' => $photo->getAlbum()->getId())) ?>">    
+                        <?= $photo->getAlbum()->getStartDateTime()->format('Y') ?>/<?= $photo->getAlbum()->getEndDateTime()->format('Y') ?>
+                    </a>        
+                </li>
+                    
+                <li>
+                    <a href="<?= $this->url('photo/album', array('album_id' => $photo->getAlbum()->getId())) ?>">
+                        <?= $photo->getAlbum()->getName() ?>
+                    </a>
+                </li>
+            
+<!--
```

Comment:
It seems more WIP than I thought. I think I should split the sass part into it's own PR, and rebase the design on top of that.